test/dartdoc_integration_test.dart

```diff
@@ -0,0 +1,235 @@
+// Copyright (c) 2015, the Dart project authors.  Please see the AUTHORS file
```

Comment:
2019?

Reply:
done, and renamed the new library in the `library` statement